src/commands/client/client_command_handlers.c

```diff
@@ -228,7 +228,7 @@ void execute_command_benchmark(const char *cmd, client_t *client,
              * and use them here so we don't spend too much of our benchmarking
              * time generating and formatting uuid keys as strings
              */
-            char key[32];
+            char key[512];
```

Comment:
@hkakutalua Let's use the `KEY_LEN` constant defined in the `client_command_handlers.c` to set the total length of characters for that key.